pom.xml

```diff
@@ -209,7 +218,7 @@
         <executions>
           <execution>
             <goals>
-              <goal>integration-test</goal>
+              <!--<goal>integration-test</goal>-->
```

Comment:
Ok, my bad. It is actually possible to skip the tests from command line (available on maven official documentation, but first googlable answers on stackoverflow are misleading). 
`mvn clean install -DskipITs`
will do the job